core/frontend/package.json

```diff
@@ -102,7 +102,8 @@
     "ldclient-js": "^2.6.0",
     "oidc-client": "^1.9.1",
     "semver": "^5.5.0",
-    "wms-capabilities": "0.4.0"
+    "wms-capabilities": "0.4.0",
+    "xml-js": "~1.6.11"
```

Comment:
@mdastous-bentley how large is this package?  The way it's currently used it will be bundled into every iModel.js frontend.

Reply:
[unpacked size 421kb](https://www.npmjs.com/package/xml-js)

Reply:
@mdastous-bentley is this being used only for tests?

Reply:
nope.

Reply:
@mdastous-bentley  - Can you modify the wms-capabilities to handle WMTS -- I think that was my original plan. - Their layout is very similar?

Reply:
@RBBentley 'wms-capabilities' the open source package we are using or WmsCapaibilities.ts?  Actually the capabilities format of WMTS and WMS are quite different:  WMS has a single hierarchy of sublayers.  WMTS a simple flat list of layers but also introduces several new concepts: TileMatrixSetLink, TimeMatrixSetLimits, TileMatrixSet and TileMatrix.

Reply:
I was referring to the open source package - was hoping that the formats would be close enough that the WMS packagecould be easily extended to handle WMTS - but I didn't look into this  at all and there may well be good reason why not.

Reply:
Analyzing the bundle now and it adds ~15.61 KB to our overall size once gzipped so fairly small percentage overall.

We do already have [xmldom](https://www.npmjs.com/package/xmldom) pulled in via the `itwin-client` that will get included into every bundle already.  It doesn't have a toJson feature but it would allow you to grab items directly out of the xml.  Would you be a able to use that or would it add a lot more manual work?

Reply:
Discussed with @mdastous-bentley and the fast-xml-parser ends up being very close in size to xml-js so he switched over to that one for the performance improvements.

We also discussed future plans to avoid this being pulled into every app, but for now they'll get this by default.

I won't go into too many details here but we discusses a bit about pulling the wms and wmts functionality (or parts of it) into a separate package that can be optional added into IModelApp.  This would mean by default the functionality wouldn't be available in iModelApp but something like the iTwin Viewer can include it, meaning most apps would then have it pre-setup for them. We can discuss this outside of this PR though...